api/tags.py

```diff
@@ -3,11 +3,15 @@
 from db.models import Tag
 from insights_connexion import responses
 from insights_connexion.db.gino import db
+from sqlalchemy import and_
 
 
 async def _get_one_tag(id):
     return await Tag.query.where(Tag.id == id).gino.first()
 
+async def _get_duplicate_tag(account_id, namespace, name, value):
+    tags = await Tag.query.where(and_(Tag.account_id == account_id, Tag.namespace == namespace, Tag.name == name, Tag.value == value)).gino.all()
+    return await Tag.query.where(and_(Tag.account_id == account_id, Tag.namespace == namespace, Tag.name == name, Tag.value == value)).gino.first()
```

Comment:
@kruai what is the `tags` variable doing here?

Reply:
Oops, that's a leftover from debugging. I'll remove it!